packages/runtime-common/realm.ts

```diff
@@ -1108,9 +1108,6 @@ export class Realm {
     requestContext: RequestContext,
   ): Response {
     try {
-      if (content.match(/^\s*$/)) {
-        throw new Error('File is empty');
-      }
```

Comment:
An empty file for `.ts` and the like isn’t invalid, so empty file detection is moved to the client.

Reply:
hmmm, i think an empty file is not technically a TS file. in order to be a TS at the very least it should export nothing. like: 
```
export {};
```
perhaps for executable files we always add at least this on file create?